onboarding/objectives_setting.md

```diff
@@ -0,0 +1,54 @@
+# Objective Setting
+
+## Probation
+
+In the first six months at Turing, new starters are in a probationary period for which they will
+set goals for review at the end of probation. 
+To learn more about the Probation process and timelines,
+see this [Mathison document](https://mathison.turing.ac.uk/page/2466).
+
+The target is for the new starter to set objectives within week 1-3 with an assessment 
+at the 5 months mark. 
+It is recommended to use the [SMART goals framework](https://en.wikipedia.org/wiki/SMART_criteria).
+
+### Example RAM Objectives for Probation
+
+These objectives were created by new RAMs in the AI for Science and Government (ASG) programme.
+
+1. Contribute to Turing research impact through stakeholder communication and engagement activities
+-  Facilitate the publication of 1-2 impact stories with the Comms team
+-  Organise workshop with external stakeholder
+
+2. Scope and kickstart 2-3 RAM engagements across the programme
+- Meet with multiple researchers in the programme to asssess readiness for RAM engagement
+- Make connections within the programme and beyond in the wider Turing network 
+
+3. Upskill on RAM, TPS, and TTW ways of working & identify opportunities to feed back
+- Open issues and begin contributing to the RAM and TTW repos
+- Co-create workshop to bring RAM or TTW practices and values to other Turing programmes and communities
+
+## Appraisal
+
+Every year, Turing runs an appraisal process which consists of objective setting for performance review and is part of any performance-related pay discussion.
+Objectives are set in September and will be reviewed again in May of the next year.
+That process consists of a self evaluation on Cezanne, line manager evaluation on Cezanne, and performance review meeting with the line manager.
+To learn more about theses processes, see the Mathison documentation on [Appraisal](https://mathison.turing.ac.uk/Interact/Pages/Section/ContentListing.aspx?subsection=3715) and [Performance Review](https://mathison.turing.ac.uk/page/2319).
```

Comment:
I really appreciate these links. However, I wonder about the stability of them. Perhaps we need to have a standing/regular review of the links to make sure they're working. (We could have done it automatically if it wasn't links directed at the Mathison site...)

Reply:
No, no. I think links are extremely helpful. Just wanted to make sure that we remember this for the future.

Reply:
Regular cadence of link checking is a the epitome of important-not-urgent tasks